pyproject.toml

```diff
@@ -28,21 +28,26 @@ nml = "nannyml.cli.cli:cli"
 nannyml = "nannyml.cli.cli:cli"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
-pandas = [
-    {version="^1.5", python=">=3.8,<3.12"}
-]
+python = ">=3.9,<3.13"
+
+numpy = "^2.0"
+
+scipy = "^1.13"
+
+pandas = "^2.2"
+
+scikit-learn = "^1.6"
```

Comment:
isn't this the very latest version released dec 9? I think ideally deps shouldn't be too constrained, as this would make it harder for users to integrate nannyml into their model monitoring.